app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,19 @@
+from .elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(
+            self,
+            nickname: str,
+            musical_instrument: str,
+            bow_level: int
+    ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def get_rating(self) -> int:
+        return 3 * self._bow_level
+
+    def player_info(self) -> str:
+        return (f"Elf ranger {self.nickname}. {self.nickname}"
+                f" has bow of the {self._bow_level} level")
```

Comment:
In the `player_info` method, the string concatenation could be improved for clarity. The repetition of `{self.nickname}` in the sentence might be unnecessary. Consider revising the sentence structure to make it more concise.